packages/samples/headless-commerce-ssr-remix/app/entry.server.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * By default, Remix will handle generating the HTTP Response for you.
+ * You are free to delete this file if you'd like to, but if you ever want it revealed again, you can run `npx remix reveal` ✨
+ * For more information, see https://remix.run/file-conventions/entry.server
+ */
+import type {AppLoadContext, EntryContext} from '@remix-run/node';
+import {createReadableStreamFromReadable} from '@remix-run/node';
+import {RemixServer} from '@remix-run/react';
+import {isbot} from 'isbot';
+import {PassThrough} from 'node:stream';
+import {renderToPipeableStream} from 'react-dom/server';
+
+const ABORT_DELAY = 5_000;
+
+export default function handleRequest(
+  request: Request,
+  responseStatusCode: number,
+  responseHeaders: Headers,
+  remixContext: EntryContext,
+  // This is ignored so we can keep it in the template for visibility.  Feel
+  // free to delete this parameter in your app if you're not using it!
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  loadContext: AppLoadContext
+) {
+  return isbot(request.headers.get('user-agent') || '')
```

Comment:
Mh, let's remove the is or isn't bot IMO.
I would not encourage implementers to lie to Lighthouse, you're more likely to get dinged by big G instead.